### DISCUSSION TITLE
GSoD'23 participation for Ruby

### DISCUSSION THREAD

**Discussion by gauriimaheshwarii** [ioquatix/ioquatix#18] [General]
Hi!
I am Gauri Maheshwari, a second-year student pursuing Mechanical and Automation Engineering at the Indira Gandhi Delhi Technical University for Women, India.

I am a technical content writer and wanted to participate in GSoD'23. Project Ruby needed documentation for an installation guide for Windows users, and I find myself an appropriate candidate to write the same. 
I wanted to know about the hiring process for technical content writers and how I could start contributing to the documentation.

I can be found here:
[Email](mailto:gaurimaheshwari009@gmail.com)
[Twitter](https://twitter.com/gaurii09)
[LinkedIn](https://www.linkedin.com/in/gaurimaheshwari/)

Any advice would be of immense help.
Thank you!

**Comment by ioquatix** [comment]
Hi, I think Ruby can always use improved documentation. It's also good to have documentation written from the first user experience perspective - the first time you try to use or understand something is a special moment which can greatly help to create documentation for other users going through the same process.

- What's your level of ability with Ruby?
- What kind of technical writing do you want to do?
- What kind of problems do you enjoy solving/writing about?

cc @burdettelamar do you have any thoughts?

**Reply by gauriimaheshwarii** [reply]
Understanding something for the first time is indeed a special moment.

- I started with Ruby around a month ago, so I am a beginner with the language. But I am willing to put in the effort to learn and upskill.
- I believe that my writing should be able to help the reader with the problem they face, and they should be clear about what to do next. It should be able to simplify things step-by-step.
- I enjoy solving problems that are not readily available on the internet and require references from multiple sources. I prefer to simplify and present them for easy understanding.
